Editorial Review (#2024)
flyteorg/flyte: rsts/concepts/basics.rst

       :text: Tasks
       :classes: btn-block stretched-link
    ^^^^^^^^^^^^
    A **Task** is any independent unit of processing. Tasks can be pure functions or functions with side-effects.

>     Each definition of a task also has associated configurations and requirements specifications.

Reviewer: ```suggestion
    Each task is also associated configurations and requirements.
```

Author: Thanks Niels. Will run the editorial review after earlier conflicts have been resolved.

    .. link-button:: divedeep-data-management
        :type: ref
        :text: How Flyte Manages Data
        :classes: btn-block stretched-link
    ^^^^^^^^^^^^

>     What is **metadata**? How are large amounts of **raw data** handled? How does data flow between tasks?

Reviewer: ```suggestion
    As a data-aware orchestration tool, Flyte only keeps track of the **metadata** passing through your tasks and workflows, keeping **raw data** within your own data stores.
```

Author: Thanks Niels. Will run the editorial review after earlier conflicts have been resolved.
